physicsnemo/__init__.py

```diff
@@ -19,4 +19,4 @@
 from .models.meta import ModelMetaData
 from .models.module import Module
 
-__version__ = "1.3.0a0"
+__version__ = "1.3.0"
```

Comment:
We shouldn't be changing the software version this commit.

Reply:
@ktangsali are you code owner on `physicsnemo/__init__.py`?  If not, we should ensure that :)